v3/shandoncodes/riot_pad/riot_pad.json

```diff
@@ -0,0 +1,25 @@
+{
+	"name": "Riotpad",
+	"vendorId": "0x7877",
+	"productId": "0x1000",
+	"keycodes": ["qmk_lighting"],
+	"menus": ["qmk_rgblight"],
```

Comment:
Please enable `RGB_TEST` as well in the info.json for your QMK keyboard since this menu assumes that is included. Otherwise it must use a custom menu.

https://github.com/the-via/reader/blob/master/src/common-menus/qmk_rgblight.ts#L56

Reply:
@Xelus22 I have added the PR to QMK's repo [here](https://github.com/qmk/qmk_firmware/pull/22118) . Should I just "ping" you here whenever it gets merged?

Reply:
@Xelus22 This have been resolved, the change was merged into [QMK master](https://github.com/qmk/qmk_firmware/pull/22118#pullrequestreview-1640940013).